README.md

```diff
@@ -101,7 +101,7 @@ version is. All in all, you must update:
 - The `rustdoc-types` dependency in `Cargo.toml` to the new version.
 - The `rust-toolchain` file to point to the new nightly version.
```

Comment:
Need to update the toolchain file as well I think.

Reply:
Done.